src/get-jwks.js

```diff
@@ -93,7 +99,9 @@ function buildGetJwks(options = {}) {
   }
 
   async function retrieveJwk(normalizedDomain, alg, kid) {
-    const jwksUri = providerDiscovery
+    const jwksUri = jwksPath
+      ? normalizedDomain + jwksPath
+      : providerDiscovery
```

Comment:
Can we split this?

Reply:
I split the string for clarity as I thought that's what you were referring to. I don't think there's any need to split the ternary statement itself, as it reads well and without ambiguity.

Reply:
It's not ambiguous, but I might be a little hard to get immediately though.
Anyway, it's not a big deal! :)